uint8 LOST_TARGET=0
uint8 CONVERGING=1
uint8 FIRE=2

uint8 data